library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java

```diff
@@ -276,7 +276,8 @@ public void onMapClick(LatLng latLng) {
 
   @Override public void onLocationPermissionsGranted() {
     //noinspection MissingPermission
-    googleMap.setMyLocationEnabled(myLocationEnabled);
+    googleMap.setMyLocationEnabled(true);
+    googleMap.getUiSettings().setMyLocationButtonEnabled(true);
```

Comment:
should we also set `myLocationEnabled = true` here?

Reply:
it sounds like we might have inconsistent state here when `myLocationEnabled` is `false` but location permissions were requested. Could that happen?

Reply:
@felipecsl ah so `mLocationEnabled` is set to true here if permissions were granted: https://github.com/airbnb/AirMapView/pull/117/files#diff-24c14ac195338f785c141cd2e0f19943R272

so I think we'll be okay